### ISSUE TITLE
Broken link for Kindlegen on the dependencies page

### ISSUE THREAD

**Issue by yurivict** [sile-typesetter/casile#164]
[Kindlegen](https://www.amazon.com/gp/feature.html?docId=1000234621)

**Comment by alerque** [comment]
Ya they have discontinued that even though it still works better than their current "replacement".

You can find the sources I use based on [the AUR package](https://aur.archlinux.org/packages/kindlegen), currently it is downloading the last known working version before they pulled it from The Internet Archive.

**Comment by yurivict** [comment]
The Arch port seems to be only for i386 Linux, and I am on FreeBSD.

But it looks like Kindlegen is an optional dependency.

**Comment by alerque** [comment]
Yes you can build this with `./configure KINDLEGEN=false` if you want to ignore than dependency. The consequence is that `casile make -- <book_id>.mobi` will fail. You should still be able to generate epub and other formats, just not mobi.

The same is true for most other dependencies, one could consider many of them optional if you don't use certain output formats or cover generation features or whatever.

Are you building FreeBSD packaging or just doing a source build for yourself? I'd be glad to help facilitate changes to make adaption to other platforms easier. I am aware this was originally cobbled together in a completely Linux (and Arch Linux at that) centric environment.

**Comment by yurivict** [comment]
I am trying to create the FreeBSD port for casille.
But I found that the generic podofo package doesn't work for casile and requires special patches.
I am not sure how to proceed.
